src/Admin.php

```diff
@@ -342,6 +355,14 @@ public static function cron_revision_cleanup($limit = 100) {
     }
   }
 
+  /**
+   * Cron event callback to delete orphan post meta.
+   */
+  public static function cron_delete_orphan_post_meta() {
+    global $wpdb;
+    $wpdb->query($wpdb->prepare("DELETE pm FROM {$wpdb->prefix}postmeta pm LEFT JOIN {$wpdb->prefix}posts p ON p.ID = pm.post_id WHERE p.ID IS NULL"));
```

Comment:
Not 100% sure if we need prepare here as we are not using any variables.

Reply:
Can be removed, yeah.